internal/operands/common-instancetypes/reconcile.go

```diff
@@ -343,3 +357,25 @@ func (c *CommonInstancetypes) reconcileVirtualMachineClusterPreferencesFuncs() [
 	}
 	return funcs
 }
+
+func ignoreObjectOwnedByVirtOperator(request *common.Request, obj client.Object) (bool, error) {
+	// During an upgrade to v0.19.0 we might encounter virt-operator attempting
+	// to reconcile the same set of common-instancetype resources. Ignore these
+	// requests to reconcile such objects once owned by virt-operator.
+	if err := request.Client.Get(request.Context, client.ObjectKeyFromObject(obj), obj); err != nil {
```

Comment:
nit: This call modifies the `obj` parameter. I would not expect this function to modify its parameters.
But we can change it in a future PR.
Can you please write a "TODO" comment here, so we don't forget about it?